src/frontend/js/modules/authentication.spec.js

```diff
@@ -71,6 +72,16 @@ describe('The esn.authentication tokenAPI service', function() {
       expect(promiseSpy).to.have.been.called;
     });
 
+    it('should call POST /api/jwt/generate', () => {
```

Comment:
The title of the testcase is duplicated with the previous one, What are you trying to test here?

Reply:
Yes, the test is duplicated too, I thought I removed it but I did not.
I will improve it.

Reply:
I removed it (because it was just an old test I duplicated and I did not successfully make a new test)

Reply:
>  I did not successfully make a new test

What makes you fail to write a new test?

Reply:
Struggling to setup promises for tests